freeplane/src/main/java/org/freeplane/view/swing/map/overview/MapOverviewConstants.java

```diff
@@ -0,0 +1,21 @@
+package org.freeplane.view.swing.map.overview;
+
+import java.awt.Color;
+
+import org.freeplane.api.LengthUnit;
+import org.freeplane.api.Quantity;
+
+public class MapOverviewConstants {
```

Comment:
Because of the reasons also valid for the Utils, I would move the constants to the classes they are related to. Introducing dedicated classes for constant values if rather needed of it allows you to hide some classes inside its package and keep the public API small. (By the way, interfaces are better for it because there you nobody expects behavior)

Reply:
Will be treated in next commits.